packages/app-root/src/middleware.js

```diff
@@ -0,0 +1,49 @@
+// Note that when internationalization is introduced to app-root,
+// this file may need refactoring to handle locale in pathname
+
+import { NextResponse } from 'next/server'
+
+export function middleware(req) {
+  /*
+    Redirect legacy PFE /about and /get-involved paths to new FEM paths
+  */
+  if (req.nextUrl.pathname.startsWith('/about/acknowledgments')) {
```

Comment:
Currently per PFE, https://www.zooniverse.org/about/acknowledgments -> "Not found", and https://www.zooniverse.org/about/acknowledgements -> loads. British English includes the extra e, which matches PFE, which I think we should match here if someone has bookmarked it?

Reply:
Ah good point, I'll add that redirect before merging!